content/en/docs/reference/command-line-tools-reference/feature-gates/index.md

```diff
@@ -0,0 +1,550 @@
+- The "Until" column, if not empty, contains the last Kubernetes release in which
+  you can still use a feature gate.
+- If a feature is in the Alpha or Beta state, you can find the feature listed
+  in the [Alpha/Beta feature gate table](#feature-gates-for-alpha-or-beta-features).
+- If a feature is stable you can find all stages for that feature listed in the
+  [Graduated/Deprecated feature gate table](#feature-gates-for-graduated-or-deprecated-features).
+- The [Graduated/Deprecated feature gate table](#feature-gates-for-graduated-or-deprecated-features)
+  also lists deprecated and withdrawn features.
+
+{{< note >}}
+For a reference to old feature gates that are removed, please refer to
+[feature gates removed](/docs/reference/command-line-tools-reference/feature-gates-removed/).
+{{< /note >}}
+
+### Feature gates for Alpha or Beta features
+
+{{< table caption="Feature gates for features in Alpha or Beta states" >}}
+
+| Feature | Default | Stage | Since | Until |
+|---------|---------|-------|-------|-------|
+| `APIListChunking` | `false` | Alpha | 1.8 | 1.8 |
+| `APIListChunking` | `true` | Beta | 1.9 | |
+| `APIPriorityAndFairness` | `false` | Alpha | 1.18 | 1.19 |
+| `APIPriorityAndFairness` | `true` | Beta | 1.20 | |
+| `APIResponseCompression` | `false` | Alpha | 1.7 | 1.15 |
+| `APIResponseCompression` | `true` | Beta | 1.16 | |
+| `APIServerIdentity` | `false` | Alpha | 1.20 | 1.25 |
+| `APIServerIdentity` | `true` | Beta | 1.26 | |
+| `APIServerTracing` | `false` | Alpha | 1.22 | 1.26 |
+| `APIServerTracing` | `true` | Beta | 1.27 | |
+| `AdmissionWebhookMatchConditions` | `false` | Alpha | 1.27 | 1.27 |
+| `AdmissionWebhookMatchConditions` | `true` | Beta | 1.28 | |
+| `AggregatedDiscoveryEndpoint` | `false` | Alpha | 1.26 | 1.26 |
+| `AggregatedDiscoveryEndpoint` | `true` | Beta | 1.27 | |
+| `AnyVolumeDataSource` | `false` | Alpha | 1.18 | 1.23 |
+| `AnyVolumeDataSource` | `true` | Beta | 1.24 | |
+| `AppArmor` | `true` | Beta | 1.4 | |
+| `CPUManagerPolicyAlphaOptions` | `false` | Alpha | 1.23 | |
+| `CPUManagerPolicyBetaOptions` | `true` | Beta | 1.23 | |
+| `CPUManagerPolicyOptions` | `false` | Alpha | 1.22 | 1.22 |
+| `CPUManagerPolicyOptions` | `true` | Beta | 1.23 | |
+| `CSIMigrationPortworx` | `false` | Alpha | 1.23 | 1.24 |
+| `CSIMigrationPortworx` | `false` | Beta | 1.25 | |
+| `CSINodeExpandSecret` | `false` | Alpha | 1.25 | 1.26 |
+| `CSINodeExpandSecret` | `true` | Beta | 1.27 | |
+| `CSIVolumeHealth` | `false` | Alpha | 1.21 | |
+| `CloudControllerManagerWebhook` | false | Alpha | 1.27 | |
+| `CloudDualStackNodeIPs` | false | Alpha | 1.27 | |
+| `ClusterTrustBundle` | false | Alpha | 1.27 | |
+| `ComponentSLIs` | `false` | Alpha | 1.26 | 1.26 |
+| `ComponentSLIs` | `true` | Beta | 1.27 | |
+ | `ConsistentListFromCache` | `false` | Alpha | 1.28 |
+| `ContainerCheckpoint` | `false` | Alpha | 1.25 | |
+| `ContextualLogging` | `false` | Alpha | 1.24 | |
+|  CRDValidationRatcheting | false | Alpha | 1.28 |
```

Comment:
```suggestion
|  `CRDValidationRatcheting` | `false` | Alpha | 1.28 |
```

Reply:
I'd prefer to save that fix for a follow-up PR @reylejano (it's wrong in the original, and I want it to be obvious that I'm keeping that part of the file the same).

That said, if folks feel the fix is essential for this to merge, I can do that.